src/data-workspace/section-form/section-form.js

```diff
@@ -51,9 +52,7 @@ export const SectionForm = ({ dataSet, globalFilterText }) => {
 
 SectionForm.propTypes = {
     dataSet: PropTypes.shape({
-        displayOptions: PropTypes.shape({
-            tabsDirection: PropTypes.oneOf(['vertical', 'horizontal']),
-        }),
+        displayOptions: PropTypes.string,
```

Comment:
not related to this ticket, but it looks like we also have some logic for parsing display options here (https://github.com/dhis2/aggregate-data-entry-app/blob/b48e1ff23e0a608f4d53a725b0ea220ab6227f48/src/data-workspace/section-form/displayOptions.js#L7), so we could probably just have one function for this parsing, and do it at the level. It looks like that getDisplayOptions gets used in the SectionFormSection child component.